src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Utilities/IRegistryKey.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Microsoft.Win32;
+using System;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Utilities
+{
+    internal interface IRegistryKey : IDisposable
```

Comment:
We should looking into SettingsStore instead of this - it's already an abstraction around the registry.